### ISSUE TITLE
Resident evil bugs graphics NTSC

### ISSUE THREAD

**Issue by Pointer1992** [devmiyax/yabause#741]
![bug RE 1 sega saturn 2](https://user-images.githubusercontent.com/6630292/82543521-f8623d00-9b53-11ea-94ce-72bd77ee97c1.jpg)
![bug RE 1 sega saturn 1](https://user-images.githubusercontent.com/6630292/82543525-f8fad380-9b53-11ea-9462-ddfc86f51d74.jpg)
![Bug RE1 sega saturn](https://user-images.githubusercontent.com/6630292/82543528-f9936a00-9b53-11ea-82f2-86806fec3c0f.jpg)

yabasanshiro-3.2.BETA2-23fb7d

**Comment by Pointer1992** [comment]
> ![bug RE 1 sega saturn 2](https://user-images.githubusercontent.com/6630292/82543521-f8623d00-9b53-11ea-94ce-72bd77ee97c1.jpg)
> ![bug RE 1 sega saturn 1](https://user-images.githubusercontent.com/6630292/82543525-f8fad380-9b53-11ea-9462-ddfc86f51d74.jpg)
> ![Bug RE1 sega saturn](https://user-images.githubusercontent.com/6630292/82543528-f9936a00-9b53-11ea-82f2-86806fec3c0f.jpg)
> 
> yabasanshiro-3.2.BETA2-23fb7d

yabasanshiro 3,2 stable not fixed

**Comment by Pointer1992** [comment]
Bug only scenario Jill
Scenario Chris full Working

**Comment by devmiyax** [comment]
Reading from cd buffer is stopped
```
cs2	: Command: playDisc 0bc0 1080 138b 0080 0052
cs2	:Seek length = 5003 - 5003 = 0/300000 0
partition number = 0 blocks = 1 blockfreespace = 199 fad = 138c playpartition->size = 800 isbufferfull = 0
cs2	: Command: getSectorNumber 0fc4 5100 0000 0000 0000
cs2	: ret: 0fc5 0300 0000 0000 0001
cs2	: Command: calculateActualSize 0f84 5200 0000 0000 0001
Cs2Area->calcsize = 1024
cs2	: Command: getStatus
cs2	: ret: 0fc5 0380 4101 0100 138c
cs2	: Command: getActualSize
cs2	: ret: 0bc5 0300 0400 0000 0000
cs2	: Command: getSectorData 0b44 6100 0000 0000 0001
cs2	: ret: 0b47 0380 4101 0100 138c
cs2	: Command: getStatus
cs2	: ret: 0b45 0380 4101 0100 138c
cs2	: Command: endDataTransfer 0b44 0600 0000 0000 0000
cs2	: ret: 0bc5 0300 0400 0000 0000
cs2	: Command: deleteSectorData 0b44 6200 0000 0000 0001
cs2	: ret: 0b45 0380 4101 0100 138c
cs2	: Command: getStatus
cs2	: ret: 0bc5 0380 4101 0100 138c
partition number = 0 blocks = 1 blockfreespace = 199 fad = 138d playpartition->size = 800 isbufferfull = 0
partition number = 0 blocks = 2 blockfreespace = 198 fad = 138e playpartition->size = 1000 isbufferfull = 0
partition number = 0 blocks = 3 blockfreespace = 197 fad = 138f playpartition->size = 1800 isbufferfull = 0
partition number = 0 blocks = 4 blockfreespace = 196 fad = 1390 playpartition->size = 2000 isbufferfull = 0
partition number = 0 blocks = 5 blockfreespace = 195 fad = 1391 playpartition->size = 2800 isbufferfull = 0
partition number = 0 blocks = 6 blockfreespace = 194 fad = 1392 playpartition->size = 3000 isbufferfull = 0
partition number = 0 blocks = 7 blockfreespace = 193 fad = 1393 playpartition->size = 3800 isbufferfull = 0
partition number = 0 blocks = 8 blockfreespace = 192 fad = 1394 playpartition->size = 4000 isbufferfull = 0
partition number = 0 blocks = 9 blockfreespace = 191 fad = 1395 playpartition->size = 4800 isbufferfull = 0
partition number = 0 blocks = 10 blockfreespace = 190 fad = 1396 playpartition->size = 5000 isbufferfull = 0
cs2	: Command: getSectorNumber 0fc4 5100 0000 0000 0000
cs2	: ret: 0fc5 0300 0000 0000 000a
cs2	: Command: calculateActualSize 0f84 5200 0000 0000 0001
Cs2Area->calcsize = 1024
cs2	: Command: getStatus
cs2	: ret: 0fc5 0380 4101 0100 1396
partition number = 0 blocks = 11 blockfreespace = 189 fad = 1397 playpartition->size = 5800 isbufferfull = 0
cs2	: Command: getActualSize
cs2	: ret: 0bc5 0300 0400 0000 0000
cs2	: Command: getSectorData 0b44 6100 0000 0000 0001
cs2	: ret: 0b47 0380 4101 0100 1397
cs2	: Command: getStatus
cs2	: ret: 0b45 0380 4101 0100 1397
cs2	: Command: endDataTransfer 0b44 0600 0000 0000 0000
cs2	: ret: 0bc5 0300 0400 0000 0000
cs2	: Command: deleteSectorData 0b44 6200 0000 0000 0001
cs2	: ret: 0b45 0380 4101 0100 1397
cs2	: Command: getStatus
cs2	: ret: 0bc5 0380 4101 0100 1397
cs2	: Command: getStatus
cs2	: ret: 0bc5 0380 4101 0100 1397
partition number = 0 blocks = 11 blockfreespace = 189 fad = 1398 playpartition->size = 5800 isbufferfull = 0
partition number = 0 blocks = 12 blockfreespace = 188 fad = 1399 playpartition->size = 6000 isbufferfull = 0
partition number = 0 blocks = 13 blockfreespace = 187 fad = 139a playpartition->size = 6800 isbufferfull = 0
partition number = 0 blocks = 14 blockfreespace = 186 fad = 139b playpartition->size = 7000 isbufferfull = 0
partition number = 0 blocks = 15 blockfreespace = 185 fad = 139c playpartition->size = 7800 isbufferfull = 0
cs2	: Command: getSectorNumber 0fc4 5100 0000 0000 0000
cs2	: ret: 0fc5 0300 0000 0000 000f
cs2	: Command: calculateActualSize 0f84 5200 0000 0000 0002
Cs2Area->calcsize = 2048
cs2	: Command: getStatus
cs2	: ret: 0fc5 0380 4101 0100 139c
cs2	: Command: getActualSize
cs2	: ret: 0bc5 0300 0800 0000 0000
cs2	: Command: getSectorData 0b44 6100 0000 0000 0002
cs2	: ret: 0b47 0380 4101 0100 139c
cs2	: Command: getStatus
cs2	: ret: 0b45 0380 4101 0100 139c
cs2	: Command: endDataTransfer 0b44 0600 0000 0000 0000
cs2	: ret: 0bc5 0300 0800 0000 0000
cs2	: Command: deleteSectorData 0b44 6200 0000 0000 0002
cs2	: ret: 0b45 0380 4101 0100 139c
cs2	: Command: getStatus
cs2	: ret: 0bc5 0380 4101 0100 139c
cs2	: Command: getStatus
cs2	: ret: 0bc5 0380 4101 0100 139c
partition number = 0 blocks = 16 blockfreespace = 184 fad = 139f playpartition->size = 8000 isbufferfull = 0
partition number = 0 blocks = 17 blockfreespace = 183 fad = 13a0 playpartition->size = 8800 isbufferfull = 0
partition number = 0 blocks = 18 blockfreespace = 182 fad = 13a1 playpartition->size = 9000 isbufferfull = 0
partition number = 0 blocks = 19 blockfreespace = 181 fad = 13a2 playpartition->size = 9800 isbufferfull = 0
partition number = 0 blocks = 20 blockfreespace = 180 fad = 13a3 playpartition->size = a000 isbufferfull = 0
partition number = 0 blocks = 21 blockfreespace = 179 fad = 13a4 playpartition->size = a800 isbufferfull = 0
partition number = 0 blocks = 22 blockfreespace = 178 fad = 13a5 playpartition->size = b000 isbufferfull = 0
partition number = 0 blocks = 23 blockfreespace = 177 fad = 13a6 playpartition->size = b800 isbufferfull = 0
partition number = 0 blocks = 24 blockfreespace = 176 fad = 13a7 playpartition->size = c000 isbufferfull = 0
partition number = 0 blocks = 25 blockfreespace = 175 fad = 13a8 playpartition->size = c800 isbufferfull = 0
partition number = 0 blocks = 26 blockfreespace = 174 fad = 13a9 playpartition->size = d000 isbufferfull = 0
partition number = 0 blocks = 27 blockfreespace = 173 fad = 13aa playpartition->size = d800 isbufferfull = 0
partition number = 0 blocks = 28 blockfreespace = 172 fad = 13ab playpartition->size = e000 isbufferfull = 0
partition number = 0 blocks = 29 blockfreespace = 171 fad = 13ac playpartition->size = e800 isbufferfull = 0
partition number = 0 blocks = 30 blockfreespace = 170 fad = 13ad playpartition->size = f000 isbufferfull = 0
partition number = 0 blocks = 31 blockfreespace = 169 fad = 13ae playpartition->size = f800 isbufferfull = 0
partition number = 0 blocks = 32 blockfreespace = 168 fad = 13af playpartition->size = 10000 isbufferfull = 0
partition number = 0 blocks = 33 blockfreespace = 167 fad = 13b0 playpartition->size = 10800 isbufferfull = 0
partition number = 0 blocks = 34 blockfreespace = 166 fad = 13b1 playpartition->size = 11000 isbufferfull = 0
partition number = 0 blocks = 35 blockfreespace = 165 fad = 13b2 playpartition->size = 11800 isbufferfull = 0
partition number = 0 blocks = 36 blockfreespace = 164 fad = 13b3 playpartition->size = 12000 isbufferfull = 0
partition number = 0 blocks = 37 blockfreespace = 163 fad = 13b4 playpartition->size = 12800 isbufferfull = 0
partition number = 0 blocks = 38 blockfreespace = 162 fad = 13b5 playpartition->size = 13000 isbufferfull = 0
partition number = 0 blocks = 39 blockfreespace = 161 fad = 13b6 playpartition->size = 13800 isbufferfull = 0
partition number = 0 blocks = 40 blockfreespace = 160 fad = 13b7 playpartition->size = 14000 isbufferfull = 0
partition number = 0 blocks = 41 blockfreespace = 159 fad = 13b8 playpartition->size = 14800 isbufferfull = 0
partition number = 0 blocks = 42 blockfreespace = 158 fad = 13b9 playpartition->size = 15000 isbufferfull = 0
partition number = 0 blocks = 43 blockfreespace = 157 fad = 13ba playpartition->size = 15800 isbufferfull = 0
partition number = 0 blocks = 44 blockfreespace = 156 fad = 13bb playpartition->size = 16000 isbufferfull = 0
partition number = 0 blocks = 45 blockfreespace = 155 fad = 13bc playpartition->size = 16800 isbufferfull = 0
partition number = 0 blocks = 46 blockfreespace = 154 fad = 13bd playpartition->size = 17000 isbufferfull = 0
partition number = 0 blocks = 47 blockfreespace = 153 fad = 13be playpartition->size = 17800 isbufferfull = 0
partition number = 0 blocks = 48 blockfreespace = 152 fad = 13bf playpartition->size = 18000 isbufferfull = 0
partition number = 0 blocks = 49 blockfreespace = 151 fad = 13c0 playpartition->size = 18800 isbufferfull = 0
partition number = 0 blocks = 50 blockfreespace = 150 fad = 13c1 playpartition->size = 19000 isbufferfull = 0
partition number = 0 blocks = 51 blockfreespace = 149 fad = 13c2 playpartition->size = 19800 isbufferfull = 0
partition number = 0 blocks = 52 blockfreespace = 148 fad = 13c3 playpartition->size = 1a000 isbufferfull = 0
partition number = 0 blocks = 53 blockfreespace = 147 fad = 13c4 playpartition->size = 1a800 isbufferfull = 0
partition number = 0 blocks = 54 blockfreespace = 146 fad = 13c5 playpartition->size = 1b000 isbufferfull = 0
partition number = 0 blocks = 55 blockfreespace = 145 fad = 13c6 playpartition->size = 1b800 isbufferfull = 0
partition number = 0 blocks = 56 blockfreespace = 144 fad = 13c7 playpartition->size = 1c000 isbufferfull = 0
partition number = 0 blocks = 57 blockfreespace = 143 fad = 13c8 playpartition->size = 1c800 isbufferfull = 0
partition number = 0 blocks = 58 blockfreespace = 142 fad = 13c9 playpartition->size = 1d000 isbufferfull = 0
partition number = 0 blocks = 59 blockfreespace = 141 fad = 13ca playpartition->size = 1d800 isbufferfull = 0
partition number = 0 blocks = 60 blockfreespace = 140 fad = 13cb playpartition->size = 1e000 isbufferfull = 0
partition number = 0 blocks = 61 blockfreespace = 139 fad = 13cc playpartition->size = 1e800 isbufferfull = 0
partition number = 0 blocks = 62 blockfreespace = 138 fad = 13cd playpartition->size = 1f000 isbufferfull = 0
partition number = 0 blocks = 63 blockfreespace = 137 fad = 13ce playpartition->size = 1f800 isbufferfull = 0
partition number = 0 blocks = 64 blockfreespace = 136 fad = 13cf playpartition->size = 20000 isbufferfull = 0
partition number = 0 blocks = 65 blockfreespace = 135 fad = 13d0 playpartition->size = 20800 isbufferfull = 0
```

**Comment by Pointer1992** [comment]
3.2.2 not fixed 
VDP2 debug info 
NGB1 and 3 glitch ?
Ngb1
8-bit(256 colors)
Tile(1H x 1V)
Plane Size = 1H x 1V
Pattern Name data size = 1 word
Character Number Supplement bit = 0
Special Priority bit = 0
Special Color Calculation bit = 0
Supplementary Palette number = 3
Supplementary Color number = 12
Plane A Address = 0007E000
Plane B Address = 0007E000
Plane C Address = 0007E000
Plane D Address = 0007E000
Screen Scroll x = 0, y = 0
Coordinate Increments x = 1.000000, y = 1.000000
Window disabled whole screen
Color Ram Address Offset = 0
Priority = 1
Color Offset A Enabled
R = 0, G = 0, B = 0
Special Color Calculation 0

NGB3
8-bit(256 colors)
Tile(1H x 1V)
Plane Size = 1H x 1V
Pattern Name data size = 1 word
Character Number Supplement bit = 0
Special Priority bit = 0
Special Color Calculation bit = 0
Supplementary Palette number = 0
Supplementary Color number = 4
Plane A Address = 0007C000
Plane B Address = 0007C000
Plane C Address = 0007C000
Plane D Address = 0007C000
Screen Scroll x = 0, y = 0
Window disabled whole screen
Color Ram Address Offset = 0
Priority = 2
Color Offset A Enabled
R = 0, G = 0, B = 0
Special Color Calculation 0

Foto by Debug VDP2 
![Bug1](https://user-images.githubusercontent.com/6630292/82747971-6f8b1180-9d9e-11ea-87f6-f92924ccaef5.jpg)
![Bug2](https://user-images.githubusercontent.com/6630292/82747972-7023a800-9d9e-11ea-8ae7-fbd3e5588e6f.jpg)

**Comment by Pointer1992** [comment]
+ info 
8-bit(256 colors)
Bitmap(512x256)
Bitmap Address = 0
Bitmap Palette Address = 10
Screen Scroll x = 0, y = 0
Coordinate Increments x = 1.000000, y = 1.000000
Window disabled whole screen
Color Ram Address Offset = 0
Priority = 3
Color Offset A Enabled
R = 149, G = 149, B = 149
Special Color Calculation 0
![Bug3](https://user-images.githubusercontent.com/6630292/82748068-56cf2b80-9d9f-11ea-91d1-9d7b56d5be45.jpg)

**Comment by Pointer1992** [comment]
3.2.3 part fix 
Scenario Jill fix 
![Jill](https://user-images.githubusercontent.com/6630292/82759723-6ece9b80-9def-11ea-9cce-152b02f8b861.jpg)
bug by foto not fix 
NBG0 mode
8-bit(256 colors)
Tile(1H x 1V)
Plane Size = 1H x 1V
Pattern Name data size = 1 word
Character Number Supplement bit = 0
Special Priority bit = 0
Special Color Calculation bit = 0
Supplementary Palette number = 0
Supplementary Color number = 12
Plane A Address = 00076000
Plane B Address = 00076000
Plane C Address = 00076000
Plane D Address = 00076000
Screen Scroll x = 0, y = -4
Coordinate Increments x = 1.000000, y = 1.000000
Window disabled whole screen
Color Ram Address Offset = 0
Priority = 7
Special Color Calculation 0


8-bit(256 colors)
Bitmap(512x256)
Bitmap Address = 0
Bitmap Palette Address = 10
Screen Scroll x = 0, y = 0
Coordinate Increments x = 1.000000, y = 1.000000
Window disabled whole screen
Color Ram Address Offset = 0
Priority = 3
Color Offset A Enabled
R = 29, G = 29, B = 29
Special Color Calculation 0


Border Color Mode = Back screen
Display Resolution = 320 x 240(NTSC)
Interlace Mode = Non-Interlace
Latches HV counter when external latch flag is read
H Counter = 0
V Counter = 248

Line Color Screen Stuff
-----------------------
Mode = Single color
Address = 05E00000

Back Screen Stuff
-----------------
Mode = Single color
Address = 05E7FFFE

Sprite Stuff
------------
Sprite Type = 3
VDP1 Framebuffer Data Format = RGB and palette
Sprite Window Enabled
Window disabled whole screen
Color RAM Offset = 0
Priority 0 = 6
Priority 1 = 5
Priority 2 = 5
Priority 3 = 5
Priority 4 = 5
Priority 5 = 5
Priority 6 = 5
Priority 7 = 5
Color Offset A Enabled
R = 29, G = 29, B = 29

**Comment by devmiyax** [comment]
Fixed in 3.3.1
